projects/aca-content/src/lib/services/content-management.service.ts

```diff
@@ -1089,4 +1090,19 @@ export class ContentManagementService {
       document.querySelector<HTMLElement>(focusedElementSelector)?.focus();
     }
   }
+
+  showFolderInformation(node: NodeEntry) {
```

Comment:
could you please write unit tests for this function?

Reply:
Added unit test